src/components/Form.tsx

```diff
@@ -85,18 +98,28 @@ export enum FormElementType {
     number = "number",
     radio = "radio",
     checkbox = "checkbox", 
+    datePicker = "date-picker"
   }
 
-  export interface State {
+export interface State {
     showPassword: boolean;
   }
 
 export const FormWithRows: React.FC<{formDetails: FormDescriptionRow}> = (props) => {
     const details =  props.formDetails;
     const classes = useStyles();
-    const[values, setValues] = React.useState<State>({
+    const [password, setPassword] = React.useState("");
+    const [passwordConfirmation, setPasswordConfirmation] = React.useState("");
+    const [email, setEmail] = React.useState("");
+    const [values, setValues] = React.useState<State>({
```

Comment:
maybe you can rename this to "showPassword" or "visibilitySettings"

Reply:
okay, thanks Cata, I will consider these comments and create new PR :)